.github/workflows/build-timestamped-master.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
```

Comment:
No need for this. This driver can be tested by publishing to dev/staging central and publish as soon as the fix is merged.

Reply:
Removed 👍